### PR TITLE
New package: EstMob.SendAnywhere version 24.3.80853

### PR DESCRIPTION
**Pull request by alexkoala**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

Replacing #149761
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/152766)

### CODE REVIEW

File: manifests/r/RakutenSymphonyKorea/SendAnywhere/24.3.80853/RakutenSymphonyKorea.SendAnywhere.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: RakutenSymphonyKorea.SendAnywhere
```
Comment (Trenly) [inline manifests/r/RakutenSymphonyKorea/SendAnywhere/24.3.80853/RakutenSymphonyKorea.SendAnywhere.installer.yaml:4]:
```suggestion
PackageIdentifier: Estmob.SendAnywhere
```
Best to match publisher from control panel

File: manifests/r/RakutenSymphonyKorea/SendAnywhere/24.3.80853/RakutenSymphonyKorea.SendAnywhere.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: RakutenSymphonyKorea.SendAnywhere
```
Comment (Trenly) [inline manifests/r/RakutenSymphonyKorea/SendAnywhere/24.3.80853/RakutenSymphonyKorea.SendAnywhere.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: Estmob.SendAnywhere
```
Best to match publisher from control panel

File: manifests/r/RakutenSymphonyKorea/SendAnywhere/24.3.80853/RakutenSymphonyKorea.SendAnywhere.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.6.0.schema.json
+
+PackageIdentifier: RakutenSymphonyKorea.SendAnywhere
```
Comment (Trenly) [inline manifests/r/RakutenSymphonyKorea/SendAnywhere/24.3.80853/RakutenSymphonyKorea.SendAnywhere.yaml:4]:
```suggestion
PackageIdentifier: Estmob.SendAnywhere
```
Best to match publisher from control panel

File: manifests/r/RakutenSymphonyKorea/SendAnywhere/24.3.80853/RakutenSymphonyKorea.SendAnywhere.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+PackageVersion: 24.3.80853
+PackageLocale: en-US
+Publisher: Rakuten Symphony Korea, Inc.
+PublisherUrl: https://send-anywhere.com/about
+PublisherSupportUrl: https://support.send-anywhere.com/hc
+PrivacyUrl: https://send-anywhere.com/terms#privacy
+PackageName: Send Anywhere
+PackageUrl: https://send-anywhere.com
+License: Proprietary
+Copyright: (c) Rakuten Symphony Korea, Inc.
```
Comment (Trenly) [inline manifests/r/RakutenSymphonyKorea/SendAnywhere/24.3.80853/RakutenSymphonyKorea.SendAnywhere.locale.en-US.yaml:14]: Can you provide a link to the Copyright?

File: manifests/r/RakutenSymphonyKorea/SendAnywhere/24.3.80853/RakutenSymphonyKorea.SendAnywhere.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,18 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: RakutenSymphonyKorea.SendAnywhere
+PackageVersion: 24.3.80853
+PackageLocale: en-US
+Publisher: Rakuten Symphony Korea, Inc.
```
Comment (Trenly) [inline manifests/r/RakutenSymphonyKorea/SendAnywhere/24.3.80853/RakutenSymphonyKorea.SendAnywhere.locale.en-US.yaml:7]:
```suggestion
Publisher: Estmob Inc.
```
This must match control panel

**Review by Trenly — CHANGES_REQUESTED** [review]
@u254254 - As you were the user who originally filed the issue around the PUA in the Korean unattended install, would you be able to verify whether downloading the installer and running it from PowerShell using `./SendAnywhereSetup.exe /S` works to install just SendAnywhere and not the PUA?

cc @SpecterShell as they were somehow able to find the NSIS script the older version of the installer used

**Comment by Trenly** [comment]
I did test in a VM set to ko-KR locale and didn't see any PUA installed, so I'm inclined to believe the ISV did remove it. However, I'd like to wait a bit to see if @u254254 responds. I'd also like to discuss this with @denelon as this is the first Flagged Package which could potentially be unflagged, and I want to be certain that it is thoroughly tested and complies with all the [WinGet policies](https://docs.microsoft.com/en-us/windows/package-manager/package/windows-package-manager-policies)

**Comment by denelon** [comment]
As long as the PUA has been removed, I'm fine bringing this package back.

**Comment by SpecterShell** [comment]
> cc @SpecterShell as they were somehow able to find the NSIS script the older version of the installer used

The part of NSIS script that installs the PUA appears to have been removed from the latest installer.

**Comment by Trenly** [comment]
[Policy] Reset Labels

**Review by Trenly — APPROVED** [review]

**Pull request merged**